jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -89,24 +89,33 @@
       <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=functional --branch=mozilla-aurora --build-revision=$REVISION --build-locale=$LOCALE --installer-url=$INSTALLER_URL</commandLine>
+      <commandLine>python -u submission.py --test-type=functional --state=running --repository=mozilla-aurora --revision=$REVISION --locale=$LOCALE treeherder_venv</commandLine>
```

Comment:
Yeah, aborting the entire test run makes sense, as long as we notice right away. 

On pf-jenkins, there were a couple of times when Treeherder submission broke (due to a quiet API change, for example) and we only noticed after a couple of days. All our tests still ran as usual, despite the Treeherder issues, which was good -- at least we knew the tests were still passing.